.github/pull_request_template.md

```diff
@@ -13,6 +13,8 @@ resolves #issue_for_this_pr
 * [ ] My changes **do not** need to be backported to a previous version
     * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
 * [ ] I have added all required tests (Unit tests, E2E tests)
+* [ ] I have added/updated all related diagnostic events and their documentation
+    * [ ] If so: Documentation issue linked to PR
```

Comment:
Should we also add an option that says `i don't have any events to log.`

Reply:
The change can follow the pattern established here: 
- [ ]   not needed
  - [ ]  Otherwise: Link/info

Reply:
Updated.